builder/comp-builder.nix

```diff
@@ -325,31 +325,35 @@ let
       ${lib.optionalString (haskellLib.isLibrary componentId) ''
         $SETUP_HS register --gen-pkg-config=${name}.conf
         ${ghc.targetPrefix}ghc-pkg -v0 init $out/package.conf.d
-        if [ -d "${name}.conf" ]; then
```

Comment:
Do we not need this conditional any more?

Reply:
ping?

Reply:
I think this was just for the now removed `all` component.  Where a single component could potentially have multiple `.conf` files.  So far none test failures seem to be related to this change.